test/integration/persistent_model_test.rb

```diff
@@ -80,6 +80,8 @@ def teardown
 
       context "with deleting" do
         should "search with simple query" do
+          omit 'Skipping test because DELETE fails with QueryParsingException[[persistent_articles] request does not support [term]]'
```

Comment:
I don't know if this is a syntax error - maybe `term` is the wrong name or not needed anymore?